src/components/events/EventParkingMap.tsx

```
@@ -0,0 +1,219 @@
+import React, { useState } from "react";
+import { Car, Navigation, MapPin, ExternalLink, ShieldCheck, Footprints, RefreshCw } from "lucide-react";
```

ESLint

[error] Replace `·Car,·Navigation,·MapPin,·ExternalLink,·ShieldCheck,·Footprints,·RefreshCw·` with `⏎··Car,⏎··Navigation,⏎··MapPin,⏎··ExternalLink,⏎··ShieldCheck,⏎··Footprints,⏎··RefreshCw,⏎`

(prettier/prettier)